pkg/controller/installation/core_controller.go

```diff
@@ -1347,6 +1308,7 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 	nodeCfg := render.NodeConfiguration{
 		K8sServiceEp:            k8sapi.Endpoint,
 		Installation:            &instance.Spec,
+		IPPools:                 ippool.CRDPoolsToOperator(currentPools.Items),
```

Comment:
The same minor behavioral change here -  we use all configured pools, not just the one in the Installation CR.

I do think this is the "correct" behavior. These pools are used to determine if the CNI plugin should assign an IPv4 and/or an IPv6 address when building the CNI config JSON. Using the full set of "enabled" IP pools in the cluster seems like the right way to do that.